app/src/main/res/layout/topic_practice_footer_view.xml

```diff
@@ -20,7 +20,8 @@
     <Button
       android:id="@+id/topic_practice_start_button"
       style="@style/StateButtonInactive"
-      android:layout_width="160dp"
+      android:layout_width="wrap_content"
```

Comment:
Will this button actually look correct if the text extends past 160dp?

Reply:
Hmm, 'Start' is rather short in English (as compared to what it might be in other languages), but it's also constrained by ConstraintLayout so I'm guessing the text will eventually be forced to wrap within the button.